### PR TITLE
Remove dependencies system

### PR DESCRIPTION
**Pull request by richibrics**
As described in #31 by @infeeeee

### CODE REVIEW

**Comment by richibrics** [comment]
This seems okay and ready to be merged, what do you think @infeeeee ?

File: IoTuring/Entity/Deployments/Cpu/settings.yaml
Comment (infeeeee) [inline IoTuring/Entity/Deployments/Cpu/settings.yaml:1]: Is this file used anywhere? I can't find any lines opening it, maybe this just remained from PyMonitorMQTT?

**Review by infeeeee — APPROVED** [review]
LGTM 🚀

My comment on `Cpu/settings.yaml` is not related to this PR.

**Comment by richibrics** [comment]
Okay I tested it and it works 🚀

**Pull request merged**